fmt/printf.h

```diff
@@ -92,9 +92,18 @@ class ArgConverter : public ArgVisitor<ArgConverter<T>, void> {
       visit_any_int(value);
   }
 
+  void visit_char(char value) {
+    if (type_ != 's' && type_ != 'S')
```

Comment:
I think that having uppercase `S` is redundant and inconsistent with other specifiers. Please leave only `s` here and below.